src/tests/test_connection.py

```diff
@@ -577,7 +588,19 @@ def test_feedback_when_switching_connection_without_alias(ip_empty, tmp_empty, c
     ip_empty.run_cell("%sql duckdb://")
 
     captured = capsys.readouterr()
-    assert "Switching to connection duckdb://" == captured.out.replace("\n", "")
+    assert "Switching to connection duckdb://" == captured.out.splitlines()[-1]
+
+
+def test_feedback_when_switching_connection_with_existing_connection(
+    ip_empty, tmp_empty, capsys
```

Comment:
please add a test for the case this message won't be displayed

Reply:
[Added!](https://github.com/ploomber/jupysql/pull/796/files#diff-ecfdf2a1f3adc925d3c32b8aa52540474a10a4160d552f281d7dbd85a0260170R606)